spec/features/add_member_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'rails_helper'
+require 'spec_helper'
+require_relative './helpers/login_helper'
+require 'selenium-webdriver'
+include LogInHelper
+
+
+feature 'Add someone to a team' do
+  scenario 'send the invitation as sender and receive it', :js=>true do
+
+    # sign in
+    log_in('student4346', 'password')
```

Comment:
I understand that may not in accord with the testing convention; but the I
think the meaning for integration test is through simulating user behavior
to test the functions. Using development database can also do that.

On Sat, Dec 5, 2015 at 9:16 PM, Yang Song <notifications@github.com> wrote:

> In spec/features/add_member_spec.rb
> <https://github.com/expertiza/expertiza/pull/640#discussion_r46765756>:
>
> > @@ -0,0 +1,45 @@
> > +require 'rails_helper'
> > +require 'spec_helper'
> > +require_relative './helpers/login_helper'
> > +require 'selenium-webdriver'
> > +include LogInHelper
> > +
> > +
> > +feature 'Add someone to a team' do
> > +  scenario 'send the invitation as sender and receive it', :js=>true do
> > +
> > +    # sign in
> > +    log_in('student4346', 'password')
>
> Hi, you need to take a look on this post:
> https://piazza.com/class/idkvxa78cn13c9?cid=403.
>
> For this kind of test you cannot assume that "student4346" and assignment
> "Ethical analysis 2" are in the testing db. When the test starts, there is
> only the data in seeds.rb in the testing db. However, the "admin" account
> is defined and you can use admin to create students' account and assignment
> before you do your test.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/expertiza/expertiza/pull/640/files#r46765756>.
>